recipe/meta.yaml

```diff
@@ -10,23 +10,35 @@ source:
   fn: {{ name }}-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
   sha256: {{ sha256 }}
+  patches:
+    ############################################################
+    # Add a patch to build on Windows.                         #
+    # xref: https://github.com/peterjc/backports.lzma/pull/25  #
+    ############################################################
+    - PR_25.patch
 
 build:
-  number: 1
-  skip: true  # [win]
-  script: python setup.py install
+  number: 2
+  script:
+    - python setup.py build_ext --compiler=mingw32                    # [win32 and py2k]
+    - python setup.py build_ext --compiler=mingw32 --define=MS_WIN64  # [win64 and py2k]
```

Comment:
The `--define=MS_WIN64` part comes from this [SO answer]( https://stackoverflow.com/a/19867426 ). Also noted it in this [comment in the upstream PR]( https://github.com/peterjc/backports.lzma/pull/25#issuecomment-353814843 ). Hopefully will be absorbed there so that we can drop it from here.

Reply:
No longer needed as the upstream PR absorbed this change. Hence dropping this in PR ( https://github.com/conda-forge/backports.lzma-feedstock/pull/7 ) where the upstream PR patch is refreshed.